public/index.html

```diff
@@ -55,6 +55,26 @@
       <div>
         <header id="root-header"></header>
         <div id="root"></div>
+        <!-- BEGIN PATCH TO SIMULATE SPA NAVIGATION -->
+        <div class="buttons-card">
+          <button
+            id="btnDashboard"
+            type="button"
+            class="dp-button button-medium primary-green"
+            onclick="window.history.pushState({}, '', '/dashboard')"
+          >
+            Dashboard
+          </button>
+          <button
+            id="btnReports"
+            type="button"
+            class="dp-button button-medium primary-green"
+            onclick="window.history.pushState({}, '', '/reports')"
+          >
+            Reports
+          </button>
+        </div>
+        <!-- END PATCH TO SIMULATE SPA NAVIGATION -->
```

Comment:
Maybe

`<div class="dp-container">
      <div class="dp-rowflex">
        <div class="col-sm-12 m-t-24">
          <button type="button" class="dp-button button-medium primary-green button--round m-r-24">
            Dashboard
          </button>
          <button type="button" class="dp-button button-medium primary-green button--round">
            Reports
          </button>
        </div>
      </div>
    </div>`

Reply:
I take some of your code and I get it: 

![image](https://user-images.githubusercontent.com/1157864/182845083-2e82e16c-0ece-47e8-8f0a-4f16107b4f6c.png)

Thanks!